### PR TITLE
change format of references

### PR DESCRIPTION
**Pull request by mosesmarin**
change format of references, add link to paper in descriptive text, set italic for quote.

### CODE REVIEW

**Review by tanzhenyu — DISMISSED** [review]

**Comment by rthadur** [comment]
@mosesmarin can you please fix ubuntu sanity build failures.

**Comment by mosesmarin** [comment]
fixed ubuntu sanity build failures.

File: tensorflow/python/keras/optimizer_v2/adam.py [hunk 1]
```diff
@@ -32,16 +32,14 @@ class Adam(optimizer_v2.OptimizerV2):
...
   requirement, invariant to diagonal rescaling of gradients, and is well suited
-  for problems that are large in terms of data/parameters'.
+  for problems that are large in terms of data/parameters*".
 
-  # References
-      See [Kingma et al., 2014](http://arxiv.org/abs/1412.6980)
-        ([pdf](http://arxiv.org/pdf/1412.6980.pdf)).
-      For AMSGrad see [Reddi et al., 2-18]
-        (https://openreview.net/pdf?id=ryQu7f-RZ)
+  For AMSGrad see [On The Convergence Of Adam And Beyond. Reddi et al., 5-8](https://openreview.net/pdf?id=ryQu7f-RZ).
```
Comment (tanzhenyu) [inline tensorflow/python/keras/optimizer_v2/adam.py]: looks like it's over 80 chars? Can you split it?

Comment (mosesmarin) [reply tensorflow/python/keras/optimizer_v2/adam.py]: line has been split

**Comment by mosesmarin** [comment]
Added commit to split long line

**Review by tanzhenyu — APPROVED** [review]

**Pull request merged**